manifests/v2/overlays/kubeflow/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Can we name this overlay: `kubeflow-platform` to make it consistent with the naming: https://www.kubeflow.org/docs/started/introduction/#what-is-kubeflow-platform.